src/main/java/windebot/History.java

```diff
@@ -0,0 +1,109 @@
+package windebot;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.Todos;
+
+/**
+ * The History class handles saving and loading the reminder list to and from a text file.
+ * It is used to persist the task data across different sessions of the application.
+ */
+
+public class History {
+    private static final String WINDE_FILE = "./src/main/java/WindeTasks.txt";
+    private static String filePath;
+
+    /**
+     * Constructs a History object with the specified file path.
+     *
+     * @param filePath The path to the file where the reminder list is stored.
+     */
+
+    History(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Constructs a History object with the default file path.
+     */
+
+    History() {
+        this.filePath = WINDE_FILE;
+    }
+
+    /**
+     * Loads the reminder list from the text file.
+     *
+     * @return An ArrayList of tasks loaded from the file.
+     */
+
+    public static ArrayList<Task> load() {
+        File file = new File(filePath);
+        ArrayList<Task> taskList = new ArrayList<Task>();
+        try {
+            if (!file.exists()) {
+                file.createNewFile();
+            }
+        } catch (IOException ioe) {
+            System.out.println("Error in creating file" + ioe.getMessage());
+        }
+        try {
+            BufferedReader br = new BufferedReader(new FileReader(filePath));
+            String line = br.readLine();
+            String complete = "X";
+            Task task;
+            while (line != null) {
+                if (line.startsWith("T")) {
+                    String[] instr = line.split(" \\| ");
```

Comment:
I like this bit of code! 👍 Maybe you can consider using a different variable name instead of `instr` and leave it in a plural form?